CoinbasePro/WebSocket/WebSocket.cs

```diff
@@ -102,23 +102,51 @@ public void Stop()
             Log.Information("WebSocket stopped");
         }
 
+
+
+        public void ChangeChannels(List<ChannelType> channelTypes)
```

Comment:
because this argument is called `channelTypes` it hides the field above. maybe name this something else. it would allow you to get rid of the `this.` below too